### PR TITLE
Document the origin of third_party files

### PR DESCRIPTION
**Pull request by mit-mit**

### CODE REVIEW

**Comment by mit-mit** [comment]
cc @renaudd can you please also give this a review?

**Review by leonsenft — APPROVED** [review]

File: pubspec.yaml [hunk 1]
```diff
@@ -9,6 +9,7 @@ environment:
 
 dependencies:
   source_span: ^1.8.0
+  path: ^1.8.0
```
Comment (leonsenft) [inline pubspec.yaml]: This should be under dev_dependencies since it's only used in tests, correct?

Comment (mit-mit) [reply pubspec.yaml]: Yes, fixed!

**Comment by mit-mit** [comment]
I fixed the pubspec dependency, fixed a spelling error, and prepared for publishing. Can you take another quick look @leonsenft  ?

**Review by leonsenft — APPROVED** [review]

**Pull request merged**